test/integration/test-amp-ad-3p.js

```diff
@@ -23,261 +23,254 @@ function createFixture() {
-            () => {
-              return iframe.contentWindow.ping.resizeSuccess != undefined;
-            },
-            null,
-            5000
+        return poll(
+          'wait for attemptChangeSize',
+          () => {
+            return iframe.contentWindow.ping.resizeSuccess != undefined;
+          },
+          null,
+          5000
+        );
+      })
+      .then(async function () {
+        lastIO = null;
+        // Ad is fully visible
+        iframe.contentWindow.context.observeIntersection((changes) => {
+          lastIO = changes[changes.length - 1];
+        });
+        await poll('wait for initial IO entry', () => {
+          return (
+            lastIO != null &&
+            lastIO.boundingClientRect.top == 1000 &&
+            lastIO.intersectionRatio == 1
           );
-        })
-        .then(async function () {
-          lastIO = null;
-          // Ad is fully visible
-          iframe.contentWindow.context.observeIntersection((changes) => {
-            lastIO = changes[changes.length - 1];
-          });
-          await poll('wait for initial IO entry', () => {
-            return (
-              lastIO != null &&
-              lastIO.boundingClientRect.top == 1000 &&
-              lastIO.intersectionRatio == 1
-            );
-          });
-          await new Promise((resolve) => {
-            setTimeout(resolve, 110);
-          });
-          lastIO = null;
+        });
+        await new Promise((resolve) => {
+          setTimeout(resolve, 110);
+        });
+        lastIO = null;
 
-          // Ad is still fully visible. observeIntersection fire when
-          // ads is fully visible with position change
-          fixture.win.scrollTo(0, 1000);
-          fixture.win.dispatchEvent(new Event('scroll'));
-          await poll('wait for new IO entry when ad is fully visible', () => {
+        // Ad is still fully visible. observeIntersection fire when
+        // ads is fully visible with position change
+        fixture.win.scrollTo(0, 1000);
+        fixture.win.dispatchEvent(new Event('scroll'));
+        await poll('wait for new IO entry when ad is fully visible', () => {
+          return (
+            lastIO != null &&
+            lastIO.boundingClientRect.top == (platform.isIos() ? 1 : 0) &&
+            lastIO.intersectionRatio == 1
+          );
+        });
+        await new Promise((resolve) => {
+          setTimeout(resolve, 110);
+        });
+        lastIO = null;
+
+        // Ad is partially visible (around 50%)
+        fixture.win.scrollTo(0, 1125);
+        fixture.win.dispatchEvent(new Event('scroll'));
+        await poll(
+          'wait for new IO entry when intersectionRatio changes',
+          () => {
             return (
               lastIO != null &&
-              lastIO.boundingClientRect.top == (platform.isIos() ? 1 : 0) &&
-              lastIO.intersectionRatio == 1
+              lastIO.intersectionRatio > 0 &&
```

Comment:
Intentional change?

Reply:
I think this is a mismatch between the two versions. This compares to previous line 239 : )

Reply:
Can confirm this line wasn't changed, it was merely re-indented. I verified that the code is the same by hiding whitespaces: https://github.com/ampproject/amphtml/pull/30029/files?w=1